manifest.yaml

```diff
@@ -259,6 +259,10 @@
   BuildScript: ./Android/Io.OpenCensus/build.cake
   TriggerPaths: [ Android/Io.OpenCensus ]
   MacBuildTargets: [ libs, nuget]
+- Name: Io.PerfMark
+  BuildScript: ./Android/Io.PerfMark/build.cake
+  TriggerPaths: [ Android/Io.PerfMark ]
+  MacBuildTargets: [ libs, nuget]
```

Comment:
Make a single target named "ci" that is dun as the default target in the cake. This is because we are going to remove the manifest really soon: https://github.com/xamarin/XamarinComponents/pull/708

Reply:
Done
Replaced libs and nuget with CI